projects/hateful_memes/README.md

```diff
@@ -1 +1,87 @@
 # Hateful Memes Dataset
+
+This folder contains configs required to reproduce results and baselines in the [Hateful Memes challange paper](https://arxiv.org/abs/2005.04790). For details on participation in the challenge, please look at our [Hateful Memes Challenge tutorial](https://mmf.readthedocs.io/en/latest/notes/hateful_memes_challenge.html).
```

Comment:
s/challange/challenge/

Reply:
Is it a bit weird to link to that tutorial already if it doesn't exist yet? Can we add this later?

Reply:
Vedanuj is adding this in a separate PR.